its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -82,6 +84,7 @@ public static Path projectDir(Path temp, String projectName) throws IOException
     FileUtils.deleteDirectory(newFolder.toFile());  // FIXME: Files instead? Recursive?
```

Comment:
CaYC: Can you delete the fixme, or change it to `ToDo` if it still makes sense? `FIXME` should not make it to `master`.

I have a weak memory that I wrote it, and the single-line thing seems to do the job so I don't have need to change it further.